flybywiresim/a32nx
# fix: Fix for vPilot (7457)

Fixes #7457

## Summary of Changes
This PR fixes an issue where it was possible but not by default, to switch between COM1 and COM2 using vPilot.

The default behavior of ACP2 has changed: no CALL push button is activated by default.

## Screenshots (if necessary)

## References

## Additional context

## Testing instructions

1. Spawn at an airport
2. Start vPilot
3. ACP 2 should not turn on any CALL light
4. Switch between CALL COM1 and 2
   - [ ] vPilot should detect the switch
   


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
